scripts/prepare-husky.js

```diff
@@ -0,0 +1,45 @@
+const { execSync } = require('child_process');
+const { existsSync } = require('fs');
+
+// ===========
+// File paths.
+// ===========
+
+const FILE_COMMIT = './.husky/pre-commit';
+const FILE_COMMIT_MESSAGE = './.husky/commit-msg';
+const FILE_HUSKY = './.husky/_/husky.sh';
```

Comment:
this could work, but if it was me I would probably doing something like

```ts
const fileCommand = [
 [''./.husky/_/husky.sh', 'npx husky install'],
 ['./.husky/pre-commit', 'npx husky add .husky/pre-commit "npx lint-staged"'],
 [''./.husky/commit-msg', 'npx husky add .husky/commit-msg "npx --no -- commitlint --edit ${1}"']
]
```

so we have a more simple way to execute command if a file is not exist